content/applications/hr/payroll.rst

```diff
@@ -108,6 +108,17 @@ The contract template forms the basis of an offer, and can be modified for speci
 employees, when necessary. If a contract template is created or modified in the *Payroll*
 application, the changes are also reflected in the *Recruitment* application.
 
+.. important::
+   To access contract templates, the :guilabel:`Salary Configurator` module **must** be installed.
+
+   To install this module, navigate to the *Apps* application. In the search bar, delete the
```

Comment:
```suggestion
   To install this module, navigate to the :menuselection:`Apps` application. In the search bar, delete the
```
when instructing the reader to "navigate" somewhere, a menuselection should be applied